tools/update_all.py

```diff
@@ -63,6 +63,7 @@ def __init__(self, name: str, version: str, *, output: str | None = None) -> Non
     Lib("Poppler", "0.18"),
     Lib("Rsvg", "2.0"),
     Lib("Secret", "1"),
+    Lib("Shumate", "1.0"),
```

Comment:
I find it a bit confusing that the library has version 1.4, but one probably wants to pass `1.0` here, [as the `api_version` is `1.0`](https://gitlab.gnome.org/GNOME/libshumate/-/blob/136480ab2dd0b55f3766126e85f9a2551e328752/meson.build#L22)

Reply:
yes, its about the api version